docs/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix.md

```diff
@@ -13,6 +13,25 @@ In this tutorial, you'll explore the creation of an **analyzer** and an accompan
 
 ## Prerequisites
 
+> [!NOTE]
+> The current Visual Studio template has a known bug in it and should be fixed in Visual Studio 2019 version 16.7. The templates will not compile unless the following changes are made:
```

Comment:
```suggestion
> The current Visual Studio template has a known bug in it and should be fixed in Visual Studio 2019 version 16.7. The template will not compile unless the following changes are made:
```

Reply:
There are multiple projects in the template that fail to compile, it is correct being plural. But I've updated it a bit more for clarity.